INFO     backend.layers.business.business:business.py:500 {'message': 'ingesting dataset', 'collection_version_id': 446afe1a-f0ee-4d01-87d2-0cc876b9b9b6, 'url': 'http://fake.url', 'current_dataset_version_id': None}
INFO     backend.layers.business.business:business.py:500 {'message': 'ingesting dataset', 'collection_version_id': 446afe1a-f0ee-4d01-87d2-0cc876b9b9b6, 'url': None, 'current_dataset_version_id': 7c347e06-d464-4e00-98ef-ffd4bb5297d6}
ERROR    processing:dataset_metadata_update.py:234 Cannot find raw H5AD artifact uri for 7c347e06-d464-4e00-98ef-ffd4bb5297d6.